tasks/main.yml

```diff
@@ -1 +1,71 @@
-# role tasks
+- name: Gather system locale facts for idempotence checks
+  command: >
+    busctl --system get-property org.freedesktop.locale1
+    /org/freedesktop/locale1 org.freedesktop.locale1
+    {{ localectl_properties | join(' ') }}
+  register: localectl_busctl_gather
+  changed_when: false
+  become: "{{ localectl_become }}"
+
+# crush the busctl returns down to a 'localectl_property: value' dict for
+# easy idempotence checks later.
+- name: Parse system locale facts
+  set_fact:
+    localectl_facts: |
+      {{ localectl_busctl_gather.stdout_lines |
+         localectl_busctl_parse(localectl_properties) }}
+  become: "{{ localectl_become }}"
```

Comment:
Does setting facts require `become`?

Reply:
No, but it doesn't require an ssh connection either, so this is pretty funny.